modules/migration-network-traffic-migration-strategies.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/planning-considerations-3-4.adoc
+
+[id="migration-network-traffic-migration-strategies_{context}"]
+= Network traffic migration strategies
+
+Each strategy for migrating network traffic for stateless applications is based on these assumptions:
+
+* Applications are deployed on the {product-title} 4.x cluster.
+* If necessary, the {product-title} 4.x router default certificate includes the {product-title} 3.x wildcard SAN.
+* Each application adds an additional route with the {product-title} 3.x host name.
+* Optional: The route with the {product-title} 3.x host name contains an appropriate certificate.
+
+Review the following strategies and select the most suitable approach:
+
+[id="big-bang-migration_{context}"]
+* "Big Bang" migration
++
+Change the {product-title} 3.x wildcard DNS record to point to the {product-title} 4.x router's virtual IP address (VIP).
+
+[id="individual-applications_{context}"]
+* Individual applications
++
+Create a new record for each application with the {product-title} 3.x FQDN/host name pointing to the {product-title} 4.x router's VIP. This record takes precedence over the {product-title} 3.x wildcard DNS record.
+
+[id="canary-style-migration-individual-applications_{context}"]
+* Canary-style migration of individual applications
+
+. Create a VIP/proxy with two back ends, the {product-title} 3.x router's VIP and the {product-title} 4.x router's VIP, for each application.
```

Comment:
What does this mean? How is the user supposed to do this? Is this documented anywhere?